content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -837,6 +837,23 @@ checking if the Job has the annotation
 this annotation from Jobs. Instead, you can recreate the Jobs to ensure they
 are tracked using Pod finalizers.
 
+### Elastic Indexed Jobs
+
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+
+From Kubernetes v1.27, Indexed Jobs can be scaled up/down up/down
+by mutating both `.spec.pompletions` and `.spec.completions` together such that 
```

Comment:
I'm assuming the first is spec.parallelism and we haven't introduced a new spec?

```suggestion
by mutating both `.spec.parallelism` and `.spec.completions` together such that 
```